install_and_cache_pkgs.sh

```diff
@@ -31,7 +31,7 @@ for package in $packages; do
 
   echo -n "  Caching to $cache_filepath..."
```

Comment:
Thanks for updating the variable references to a safe syntax. Could you standardize these variables as well?